mysite/api/tests.py

```diff
@@ -816,3 +817,484 @@ def test_get_one_user_non_existent(self):
         self.assertEqual(response.status_code, 404)
         expected_response = {"detail": "Not found."}
         self.assertEqual(expected_response, json.loads(response.content))
+
+
+class UserPostViewTests(TestCase):
+    def test_post_user_successfully(self):
+        """
+        Create a new user without errors
+        """
+        user_data = {
+            "username": "test",
+            "email": "test@gmail.com",
+            "password": "1234",
+            "first_name": "Mr. Test",
+            "last_name": "Unit",
```

Comment:
Maybe this can help you creating fake realistic data https://github.com/joke2k/faker

Reply:
That would be nice. Do you think I should add it in the features strictly related to training? If not, I will add it to a list of features to add for fun after training to make the project better. Thanks!

Reply:
I leave that up to you